cmake/external_mlir.cmake

```diff
@@ -42,10 +42,23 @@ execute_process(COMMAND "${CMAKE_COMMAND}" --build . --target ext_mlir_llvm
 execute_process(COMMAND "${CMAKE_COMMAND}" --build . --target ext_mlir
                 WORKING_DIRECTORY "${MLIR_PROJECT_ROOT}")
 
-# point find_package to the pre-built libs
-set(LLVM_DIR ${MLIR_LLVM_ROOT}/build/lib/cmake/llvm)
+# Enable modules for LLVM.
+set(LLVM_CMAKE_PATH "${MLIR_BUILD_DIR}/lib/cmake/llvm"
+    CACHE PATH "Path to LLVM cmake modules")
+list(APPEND CMAKE_MODULE_PATH "${LLVM_CMAKE_PATH}")
+include(AddLLVM)
+
+# Enable LLVM package, definitions and env vars.
+find_package(LLVM REQUIRED CONFIG)
```

Comment:
We need `LLVM_DIR` set here, otherwise find_package will look else where and might use the system's LLVM instead, if present.

Reply:
Ok, so can we rename `LLVM_CMAKE_PATH` to `LLVM_DIR` and use that one only? They were both the same and used indistinctly.

Reply:
Everything work ok with and without `LLVM_DIR` with and without MLIR enabled but I replaced `LLVM_CMAKE_PATH` with `LLVM_DIR`. Please, let me know if I misunderstood something.